Yoast/wordpress-seo
# Remove unused images

## Summary
* Removed all files from the /images folder which had no usages.
#10360 


This PR can be summarized in the following changelog entry:

* Removed unused images

## Relevant technical choices:

* Deleted all unused images

## Test instructions

This PR can be tested by following these steps:

* Go to the wordpress-seo directory and check if the deleted images had no usages in the wordpress-seo directory.

## UI changes

## Documentation

## Quality assurance
